File: Java/Arrays/reverse.java
public class reverse{
    public static void reverseArray(int Array[]) {
        int start, end;
        int temp;
        start = 0;
        end = Array.length - 1;
        while (start < end) {
            temp = Array[start];
            Array[start] = Array[end];
            Array[end] = temp;
            start++;
            end--;
        }
    }
}

File: C++/Arrays/reverse.cpp
void reverse(int array[], int size)
{
    int start = 0;
    int end = size - 1;
    while (start < end)
    {
        int temp = array[start];
        array[start] = array[end];
        array[end] = temp;
        start++;
        end--;
    }
}
